news/2019-03-31-beatmap-spotlights-winter-2019.md

```diff
@@ -0,0 +1,308 @@
+
+Don't stay up late while playing this mapset, or you might experience some sleepless days!
+
+[![](/wiki/shared/news/2019-03-31-beatmap-spotlights-winter-2018/osu/amsterdam.jpg)](https://osu.ppy.sh/beatmapsets/703956)
+
+[appleeaterx](https://osu.ppy.sh/users/2407160) has always been an amazing mapper with their simple but effective maps. "Amsterdam" shows how much apple is still improving at mapping. The map has a more Russian inspired taste to it, using anti-jumps and a heavy abuse of slider leniency. However, any apple fan can still see that it's his unique style that we all fall in love with. Don't take this map lightly either, with the high OD and HP values, it becomes a challenge in itself.
+
+[![](/wiki/shared/news/2019-03-31-beatmap-spotlights-winter-2018/osu/colors.jpg)](https://osu.ppy.sh/beatmapsets/317439)
+
+[wa_](https://osu.ppy.sh/users/1572001) makes a return to the ranked section with their amazing set of "Colors sasakure.UK Futurelogic Remix" by dj TAKA. This map will really test your skills with spaced streams, stream jumps, and 1/4 jumps! It's also a massive feat to see 1 mapper mapping all the difficulties in a 4 minute song. Overall, this is a very high effort set that you'll be feeling blue when you play it!
+
+[![](/wiki/shared/news/2019-03-31-beatmap-spotlights-winter-2018/osu/saisaki.jpg)](https://osu.ppy.sh/beatmapsets/858075)
+
+Not afraid to accentuate even the tiniest of changes in the song, [cRyo\[iceeicee\]](https://osu.ppy.sh/users/835998) & co. delivers a bombastic take on Reol's song Saisaki. The shift between technical styled mapping for the EDM aspects of the song to the careful arrangements in the melody focused instrumentals and passionately delivered vocals this whole set will not fail to deliver an exhilarating experience. 
+
+Each map is in itself tied together gracefully enriching the song taking it to greater heights, and with a generous spread of difficulty the set is giving every player a change to to enjoy this masterpiece!
+
+[![](/wiki/shared/news/2019-03-31-beatmap-spotlights-winter-2018/osu/kyoumen.jpg)](https://osu.ppy.sh/beatmapsets/764481)
+
+[Smokeman](https://osu.ppy.sh/users/2140676) has always been a master of capturing the beauty in the songs he maps. His rendition of "Kyoumen no Nami [Orchestra Ver.] is no different. The map perfectly matches the very serene parts of the song with low density and low slider velocity. Then explodes with a quicker slider velocity and more difficult movements in the chorus which truly make the map shine. 
+
+Don't forget to look at the wondrous storyboard made by [LowAccuracySS](https://osu.ppy.sh/users/7306763) & [Darky1](https://osu.ppy.sh/users/3372928)!
+
+[![](/wiki/shared/news/2019-03-31-beatmap-spotlights-winter-2018/osu/bohemian.jpg)](https://osu.ppy.sh/beatmapsets/912844)
```

Comment:
missing icekalt in creators

Reply:
That is something I can't fix now. I rather not get a BSOD from photoshop